README.md

```diff
@@ -28,7 +28,7 @@ Seed has a batteries-included approach. This means less time writing boilerplate
 - It's newer. It's harder to find support outside of Discord.
 - WebAssembly is newer with less support. [Browser compatibility is at 92.9%](https://caniuse.com/?search=webassembly).
 - Pre-built components are rare. You will likely have to roll your own components such as date pickers.
-- No server-side rendering yet [(#232)](#232).
+- No server-side rendering yet [https://github.com/seed-rs/seed/issues/232](#232).
```

Comment:
I think it should be the other way around: `[#232](https://github.com/seed-rs/seed/issues/232)` and I'd also keep the `(..)` brackets